picard/const/__init__.py

```diff
@@ -17,14 +17,14 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+# Install gettext "noop" function in case const.py gets imported directly.
+import builtins
 import os
-import sys
 
 from PyQt5.QtCore import QStandardPaths
+
 from picard import PICARD_APP_NAME
 
-# Install gettext "noop" function in case const.py gets imported directly.
-import builtins
```

Comment:
Even though moving it a top respects overall style, in this case, it was making more sense for it to be near its use, that's `builtins` dict overriding, not a usual import.